clusters/hosted-mgmt/hive/pools/rh-ecosystem-edge/ocp-4-17-0_cluster-pool.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: rh-ecosystem-edge-4-17-cluster-pool
+  namespace: rh-ecosystem-edge-cluster-pool
+  labels: # architecture, cloud, owner, product, version are used to filter out a pool when a job claims a cluster
+    architecture: amd64
+    cloud: aws
+    product: ocp
+    owner: rh-ecosystem-edge
+    version: "4.17"
+    version_lower: 4.17.0-0 # optional: lower bound for automatically updated imageset; required if version_upper is set
+    version_upper: 4.18.0-0 # optional: upper bound for automatically updated imageset; required if version_lower is set
+spec:
+  baseDomain: ecoeng-edgeinfra-ci.devcluster.openshift.com # the base domain to install the cluster
+  imageSetRef:
+    name: ocp-release-4.17.0-x86-64-for-4.17.0-0-to-4.18.0-0 # the name of the imageSet which determines the image to install the cluster; will be automatically updated if `version_*` labels are set
+  installConfigSecretTemplateRef:
+    name: rh-ecosystem-edge-install-config # the name of the secret with an installation config for the installer
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: aws-credentials # the name of the secret with the credentials of the cloud account
+      region: us-east-1
+  pullSecretRef:
+    name: pull-secret
+  size: 2 # the number of clusters that Hive should keep provisioned and waiting for use.
+  maxSize: 4 # the maximum number of clusters that can exist at the same time.
```

```suggestion
  size: 6 # the number of clusters that Hive should keep provisioned and waiting for use.
  maxSize: 9 # the maximum number of clusters that can exist at the same time.
```

This pull is the bigger one, therefore, it should have the same size of what we had in 4.15 (and 4.15 needs to be scaled down to the same size of 4.14).